src/cdomains/arrayDomain.ml

```diff
@@ -693,55 +781,79 @@ module FlagConfiguredArrayDomain(Val: LatticeWithSmartOps) (Idx:IntDomain.Z):S w
 struct
   module P = PartitionedWithLength(Val)(Idx)
   module T = TrivialWithLength(Val)(Idx)
+  module U = Unroll(Val)(Idx)
```

Comment:
Is there a particular reason why we do not have a `WithLength` variant of `Unroll`? As it currently stands, if I enable `Unroll`, I will get warnings about out-of-bounds accesses everywhere, which is not really desirable.

Reply:
Fixed on 3ff05ea751b96b5b94d9f92a2ca6c26f129ae82b